web/themes/custom/recipe_theme/src/sass/block/_block.scss

```diff
@@ -0,0 +1,18 @@
+.your_recipes {
+  h2 {
+    display: flex;
+    justify-content: center;
```

Comment:
If the sole purpose here is to align this element to the center then `text-align: center;` is enough. Flex is overkill just for center aligning text.